cypress/integration/userCanSeeTitle.feature.js

```diff
@@ -0,0 +1,13 @@
+describe("Free Article view:", () => {
```

Comment:
Article?

Reply:
Made the change, thanks for catching that!